src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddleware.cs

```diff
@@ -192,6 +195,27 @@ private Task DisplayException(HttpContext context, Exception ex)
 
         private Task DisplayRuntimeException(HttpContext context, Exception ex)
         {
+            var endpoint = context.Features.Get<IEndpointFeature>()?.Endpoint;
+
+            EndpointModel endpointModel = null;
+            if (endpoint != null)
+            {
+                endpointModel = new EndpointModel();
+                endpointModel.DisplayName = endpoint.DisplayName;
+
+                if (endpoint is RouteEndpoint routeEndpoint)
+                {
+                    endpointModel.RoutePattern = routeEndpoint.RoutePattern.RawText;
+                    endpointModel.Order = routeEndpoint.Order;
+
+                    var httpMethods = endpoint.Metadata.GetMetadata<IHttpMethodMetadata>()?.HttpMethods;
```

Comment:
`Metadata.GetMetadata` sounds a bit smurfy. Is it too late to unsmurf it? 🤔

Reply:
`Meta.GetData()` :trollface:

Reply:
> Is it too late to unsmurf it?

It shipped in 2.2. It probably could have been `Get<T>()` (and there would also be a `GetOrdered<T>()` mirror). It won't be a commonly used API so not that big a deal.